### PR TITLE
Updated 3 learning paths to use Keil Studio

### PR DESCRIPTION
**Pull request by RonanSynnottArm**
Before submitting a pull request for a new Learning Path, please review [Create a Learning Path](https://learn.arm.com//learning-paths/cross-platform/_example-learning-path/)
- [x] I have reviewed Create a Learning Path

Please do not include any confidential information in your contribution. This includes confidential microarchitecture details and unannounced product information. No AI tool can be used to generate either content or code when creating a learning path or install guide.

- [x] I have checked my contribution for confidential information

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the Creative Commons Attribution 4.0 International License.

### CODE REVIEW

File: content/learning-paths/microcontrollers/asm/setup_mdk6.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+Select `CMSIS` from the Extensions icon list. You will see the project structure.
+
+Move cursor over the top-level `Project`, and click on `Manage Software Components`. This is the view to add CMSIS Software Packs to your project.
+
+Select `CMSIS > Core` and `Device > Startup` (these are likely selected by default).
+
+Close this view to save.
+
+## Configure debug with the FVP
```
Comment (annietllnd) [inline content/learning-paths/microcontrollers/asm/setup_mdk6.md:63]: Tried for VSCode on Mac, using the FVPs-on-MacOS, where it builds successfully. On running the Debugger: 
```
Failed to start model 'FVP_MPS2_Cortex-M4'. Please check that the model is installed correctly and its installation directory is included in your PATH environment variable
```
I verified that this was indeed added to the path, and runs fine when launched in the terminal.

Not sure if there's anything here that can be clarified, for either Mac or Windows.

**Review by annietllnd — COMMENTED** [review]
Hi Ronan! I seem to have some issues with getting VS Code on Mac and for Windows to pick up the FVPs. Therefore I get stuck in the debugging step, and we might have to come back to the CMSIS-RTX ones. I found some things that could be investigated in the meantime, hope this helps.

File: content/learning-paths/microcontrollers/asm/setup_mdk6.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+Locate `vcpkg-configuration.json` within the project. This file defines the components used.
+
+Right-click on this file, and select `Configure Arm Tools Environment` to open the configuration panel.
+
+From the various pull-downs, ensure that the most up to date versions of the following are selected:
+
+* Arm CMSIS-Toolbox
+* Arm Compiler for Embedded
+* Arm Debugger
+* Arm Virtual Hardware for Cortex-M based on Fast Models
```
[the remark this answers is not in the document]
Comment (RonanSynnottArm) [reply content/learning-paths/microcontrollers/asm/setup_mdk6.md:43]: Cannot replicate

File: content/learning-paths/microcontrollers/asm/setup_mdk6.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        .ANY (+RW +ZI)
+    }
+    ARM_LIB_STACK 0x20020000 EMPTY 0x4000 {}
+}
+```
+Close to save.
+
+## main.c
+
+The project automatically creates `main.c` within the `Source Files` group. We will modify this in the next section.
```
[the remark this answers is not in the document]
Comment (RonanSynnottArm) [reply content/learning-paths/microcontrollers/asm/setup_mdk6.md:101]: Done

File: content/learning-paths/microcontrollers/asm/coding.md [hunk 4]
```diff
@@ -120,12 +122,14 @@ To use the conditional `BLS` branch instruction, reduce by one the value `r1` is
 
 ## Build the example
 
-Save all files, and click the `Build` button (`F7`).
+Save all files, and click the `Build` icon.
```
[the remark this answers is not in the document]
Comment (RonanSynnottArm) [reply content/learning-paths/microcontrollers/asm/coding.md:125]: Cannot replicate

File: content/learning-paths/microcontrollers/cmsis-dsp/mdk6.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+Locate `vcpkg-configuration.json` within the project. This file defines the components used.
+
+Right-click on this file, and select `Configure Arm Tools Environment` to open the configuration panel.
+
+From the various pull-downs, ensure that the most up to date versions of the following are selected:
+
+* Arm CMSIS-Toolbox
+* Arm Compiler for Embedded
+* Arm Debugger
+* Arm Virtual Hardware for Cortex-M based on Fast Models
```
[the remark this answers is not in the document]
Comment (RonanSynnottArm) [reply content/learning-paths/microcontrollers/cmsis-dsp/mdk6.md:53]: Cannot replicate

File: content/learning-paths/microcontrollers/asm/setup_mdk6.md (hunk 1 above)
Comment (RonanSynnottArm) [reply content/learning-paths/microcontrollers/asm/setup_mdk6.md:63]: Cannot replicate, but expected if you were not able to configure project to use FVP library in previous steps.

**Pull request merged**